website/src/pages/Blog/styles.ts

```diff
@@ -3,12 +3,20 @@ import { makeStyles, Theme } from "@material-ui/core/styles";
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     position: "relative",
-    backgroundImage: `url(${"/assets/images/faq_background.png"})`,
+    backgroundImage: "url(/Images/png/faq_background.png)",
```

Comment:
will have to change `Image` to `image` once @rakeshPRaghu 's PR (https://github.com/openebs/website/pull/121) will be merged

Reply:
Yes, once Rakesh's PR is merged, will make this change